go/memcached-operator/go.mod

```diff
@@ -3,36 +3,37 @@ module github.com/operator-framework/operator-sdk-samples/go/memcached-operator
 go 1.13
```

Comment:
Just missing fix the go project.

```
$ kubectl get all -n memcached 
NAME                                     READY   STATUS             RESTARTS   AGE
pod/memcached-operator-97d84cd68-jzngb   0/1     CrashLoopBackOff   3          95s

NAME                                 READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/memcached-operator   0/1     1            0           95s

NAME                                           DESIRED   CURRENT   READY   AGE
replicaset.apps/memcached-operator-97d84cd68   1         1         0       95s

```

Reply:
Fixed this one, Please take a look again.